docs/app/pages/components/components-sections/input-controls/typeahead/snippets/app.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, ElementRef, HostListener, Inject } from '@angular/core';
+import 'chance';
+import { TypeaheadOptionEvent } from '@ux-aspects/ux-aspects';
+
+@Component({
+    selector: 'app',
+    templateUrl: 'app.component.html',
+    styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+
+    tagDocumentationRoute: string;
```

Comment:
This variable is only used in our docs site so doesn't need to be included in the sample code